docs/description/image-sizes.rst

```diff
@@ -25,6 +25,22 @@ Recommended size (aspect) to fit both usage::
 
     750 x 371
 
+You can scale picture, saving proportion.
+
 See also
 
 * :doc:`Preview module on App Store <./app-store-preview>`
+
+Adjust chromium window size script
```

Comment:
Move it to a separate page similar to 

      * :doc:`Preview module on App Store <./app-store-preview>`

Let current page be only for infromation about sizes